SPECS/libreswan/CVE-2023-38710.patch

```diff
@@ -0,0 +1,212 @@
+diff --show-c-function -Naur a/programs/pluto/ikev2_create_child_sa.c b/programs/pluto/ikev2_create_child_sa.c
```

Comment:
NIT: Can we add the patch header (if available) for these patches as well so that we maintain the original authorship.

Reply:
Libreswan distributes their packages on CVE advisory pages without attributing authorship (e.g. https://libreswan.org/security/CVE-2023-38710/)